### PR TITLE
#27500 Prepare CustomerImportExport module Tests for PHPUnit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php [hunk 1]
```diff
@@ -181,16 +205,16 @@ protected function _getModelDependencies()
      */
     public function getWebsites($withDefault = false)
     {
-        $websites = [];
+        $websites = $this->_websites;
```
Comment (ihor-sviziev) [inline app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php]: Looks like not correct change, but originally it wasn’t removing anything either. I think it’s better to to change it

File: app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/CustomerTest.php [hunk 2]
```diff
@@ -142,16 +159,16 @@ protected function _getModelDependencies()
      */
     public function getWebsites($withDefault = false)
     {
-        $websites = [];
+        $websites = $this->_websites;
```
Comment (ihor-sviziev) [inline app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/CustomerTest.php]: The same here

File: app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/CustomerTest.php [hunk 3]
```diff
@@ -165,7 +182,7 @@ public function getWebsites($withDefault = false)
      */
     public function getStores($withDefault = false)
     {
-        $stores = [];
+        $stores = $this->_websites;
```
Comment (ihor-sviziev) [inline app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/CustomerTest.php]: And here

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php (hunk 1 above)
Comment (lbajsarowicz) [reply app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php]: Yeah, whole block 
```php
        if (!$withDefault) {
            unset($websites[0]);
        }
```

should be just removed, the assignment should be still `$websites = [];`

Comment (ihor-sviziev) [reply app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php]: As we discussed in Slack - please just revert these changes in order to reduce the scope of the PR. Let's fix it separately

Comment (lbajsarowicz) [reply app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php]: Then the tests are going to fail. As it's trying to remove non-existing item.
As these methods are not being used anywhere, I'll just remove them and I hope that @magento/community-engineering-team  won't try to keep dead code.

--- EDIT

I'm wrong. It's being used in callback. :facepalm:  Will try to find other solution.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7443](https://jira.corp.magento.com/browse/ENGCOM-7443) has been created to process this Pull Request

**Pull request merged**